# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pingus/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./src/worldmap/graph.hxx.orig	2005-03-23 23:43:41.541822952 -0400
+++ ./src/worldmap/graph.hxx	2005-03-23 23:44:49.220534224 -0400
@@ -22,6 +22,9 @@
 
 #include <algorithm>
 #include <vector>
+#include <ostream>
+#include <istream>
+#include <iostream>
 
 namespace WorldMapNS {
 
